tcms/testplans/helpers/email.py

```diff
@@ -31,4 +31,4 @@ def get_plan_notification_recipients(plan):
 
     # don't email author of last change
     recipients.discard(getattr(plan.history.latest().history_user, 'email', ''))
-    return list(filter(lambda e: bool(e), recipients))
+    return list(filter(lambda e: bool, recipients))
```

Comment:
This is wrong, I think you should remove the entire first parameter of `filter()`. However you should write a small unit test for this method to make sure it doesn't change behavior.

Reply:
Drop this commit, already fixed by @ivo0126. The rest of this PR should be good to go.